### ISSUE TITLE
Revamp test

### ISSUE THREAD

**Issue by JunoNgx** [JunoNgx/justjot-frontend#33]
The test suite is atrocious due to incorrect use of element labels. Elements should have `aria-label` or `aria-labelledby` correctly and these should be used to located the elements in the tests.
